CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
  
 ### Changed
 
+## [2.20.1] - 2022-11-24
+### Added
+- xfixes feature
+
```

Comment:
Could you possibly move this into unreleased? I'll create a 2.20.1 release after merging, but I'd like to keep it a separate commit

Reply:
Same with the 2.20.1 change in Cargo.toml 😄

Reply:
Done.